tests/query/transactions/test_query_transactions.py

```diff
@@ -0,0 +1,36 @@
+import allure
+import iroha
+
+from tests import client,account_id
+from tests.helpers import generate_public_key
+
+
+@allure.label("sdk_test_id", "query_all_transactions")
+def test_query_all_transactions():
+    with allure.step('WHEN client queries all transactions'):
+        all_txs = client.query_all_transactions()
+    with allure.step('THEN there should be some transactions present'):
+        assert len(all_txs) > 0, "No transactions found in the system"
+
+
+@allure.label("sdk_test_id", "query_all_transactions_by_account")
+def test_query_all_transactions_by_account(
+        GIVEN_registered_account):
```

Comment:
I think it's logically better to also use `GIVEN_registered_transaction` in this test.